backend/models.py

```diff
@@ -1,3 +1,66 @@
-# from django.db import models
+from django.db import models
 
 # Create your models here.
+
+
+class Faculty(models.Model):
+    name = models.CharField(max_length=150)
+
+
+class FieldOfStudy(models.Model):
+    faculty = models.ForeignKey(Faculty, on_delete=models.CASCADE)
+    name = models.CharField(max_length=150)
+    degree = models.IntegerField()
+
+
+class Yearbook(models.Model):
+    field_of_study = models.ForeignKey(FieldOfStudy, on_delete=models.CASCADE)
+    begin = models.DateField()
+    end = models.DateField()
+    available_capacity = models.IntegerField()
+
+
+class Student(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    date_of_birth = models.DateField()
+    gender = models.CharField(max_length=50)
+    year_of_exam = models.IntegerField()
+    city = models.CharField(max_length=80)
+
+
+class Graduate(models.Model):
+    student = models.ForeignKey(Student, on_delete=models.CASCADE)
+    yearbook = models.ForeignKey(Yearbook, on_delete=models.CASCADE)
+    grade = models.FloatField()
+
+
+class Recruitment(models.Model):
+    field_of_study = models.ForeignKey(FieldOfStudy, on_delete=models.CASCADE)
+    year = models.IntegerField()
+    round = models.IntegerField()
+
+
+class RecruitmentResult(models.Model):
```

Comment:
Tutaj chyba dałbym jeszcze opcje, że ktoś by się dostał, ale jednak stwierdził że nie idzie. Co do stanu 'Waiting' to chyba bym go nie uwzględniał, bo w sumie to my chyba te rekrutacje trakutujemy tak, że albo się dostajesz albo nie i możesz się nie dostać w pierwszym cylku i wtedy będziesz miał rejected a w drugim cylku zostaniesz zaakceptowany i potwierdzisz to będziesz miał accepted

Reply:
taak, ale jak mamy aktualną rekrutację, to nie znamy jeszcze wyników i to chyba trzeba jakoś zapisać

Reply:
Właśnie to mnie zastanawia, bo nawet chyba przy aktualnej rekrutacji będzie to wyglądało tak, że po każdej rundzie dostajemy dane i tam będzie binarnie, że ktoś się dostał a ktoś nie. Nie wiem sam w sumie jak działają do końca te procesy, tzn, w każdej rundzie próg jest inny chyba, bo na przykład jest kierunek na 20 osób i tyle się dostaje w pierwszej rudzie. Więc my dostajemy coś takiego, że wynik tych 20 osób jest jako accepted a reszty jako rejected, bo się nie dostali w sumie. Potem się okazuje, że 7 osób nie chciało jednak iść na ten kierunek, więc w drugim cyklu wskakuje jakieś kolejne 7 osób, które miały rejected, ale teraz są już zaakceptowane. Swoją drogą nie wiem czy my właśnie takie informacje będziemy dostawać, że ktoś się dostał, czy mamy raczej sami to wyliczać jakoś? Ale wydaje mi się, że to ma być bardziej do analizy post factum, więc chyba możemy założyć, że dostajemy informacje o tym kto się dostał, a kto nie.